hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -248,6 +248,16 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
                 </div>
             </div>
 
+            <div class="form-group" ng-show="operation === 'ADD'">
+                <label for="mode" class="col-md-3 control-label">Initial state</label>
+                <div class="col-md-9">
+                    <select required class="form-control" id="initialState" name="initialState" ng-model="subscription.state">
+                        <option value="ACTIVE" selected="selected">ACTIVE</option>
```

Comment:
I think that using `selected="selected"` with `ng-model` is not necessary.

Reply:
But looks explicit, so no harm 😉

Reply:
Ok, but what if someone changes initial `state` in factory? It will be misleading, unless also changed. Other `select`s rely only on `ng-model`.

Reply:
We can add a comment in a factory.

Reply:
Actually I added it before I found out there's a Factory file. I think it's not needed and can be safely removed.